### ISSUE TITLE
During first-time app creation on begin.com consider clarifying when production goes online

### ISSUE THREAD

**Issue by filmaj** [smallwins/begin-issues#12]
**Is your feature request related to a problem? Please describe.**
Yes! First time building an app on begin.com, after the app built initially, the staging site worked great and as expected but clicking the production site yields a generic page that was much different from the staging site. It turns out it's not until I edited my app repo, and a second build went out to staging, that the tutorial dialog on begin.com told me that I had to manually push the button to deploy to production. This confused me.

**Describe the solution you'd like**
Perhaps, then, the first tutorial dialog should clarify that pushing to production requires an additional step? In the first popup, "production" is linked and I clicked on it expecting to see something like what was live on staging.

**Comment by ryanblock** [comment]
Hmm, possibly a copy issue? This is what it says now:
<img src="https://user-images.githubusercontent.com/200964/59225773-710bfb80-8b86-11e9-885c-e69ab52661e1.png" width=500>

Would it be sufficient to say something more like:
```
Run a production deploy of your new Begin app to update this environment!

Learn more about getting started with Begin here.
```

Or do you feel just needs clarification in both the Begin interface?

**Comment by filmaj** [comment]
I totally didn't see the 'awaits deployment' bit.

Not sure if possible, but what about greying out / disabling the link to the production environment in the left-hand-side of the UI until you get your first production deployment out?

**Comment by ryanblock** [comment]
I like that. Or perhaps if you click, presenting a modal of some kind could be helpful?

**Comment by filmaj** [comment]
Yeah, that makes sense. The key here is that while there is no production version deployed (i.e. version 0.0.0 is "live"), that link should behave differently compared to when 0.0.1 or any future version gets published to production.
